moped-database/metadata/tables.yaml

```diff
@@ -2372,6 +2375,13 @@
         table:
           schema: public
           name: moped_project_types
+  - name: moped_proj_milestones
+    using:
+      foreign_key_constraint_on:
+        table:
+          name: moped_proj_milestones
+          schema: public
+        column: project_id
```

Comment:
my script does *not* make use of this relationship, but it would have been nice to have it in place for this task.